docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -187,6 +187,11 @@ To enable consistent hashing for a backend:
 
 `nginx.ingress.kubernetes.io/upstream-hash-by`: the nginx variable, text value or any combination thereof to use for consistent hashing. For example `nginx.ingress.kubernetes.io/upstream-hash-by: "$request_uri"` to consistently hash upstream requests by the current request URI.
 
+"subset" hashing can be enabled using 
```

Comment:
can you add a brief info or link to another pager explaining what subset consistent hashing is?